docs/section21/Rule21-9.md

```diff
@@ -0,0 +1,38 @@
+
+# Boiler - Rule 21-9  
+
+**Rule ID:** 21-9  
+**Rule Description:** When baseline building includes boilers, Hot Water Pump Power = 19W/gpm.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+
```

Comment:
should address this applicability check for purchased heating, as commented on other RDS.
two issues
1. we will probably not be implementing the RDS for 21-1, so should have another way to verify this applicability check.
2. This statement also does not clarify that a building doesn't have purchased heating. For that we should probably have some standard schema checks for external fluid source

Reply:
Added Rule 21-1 as applicability check.

Reply:
See note above on applicability

Reply:
Same comment on 21-7 etc., that applicability should be `if Rule-21-1 == "UNDERTERMINED"